lib/diagnostic.js

```diff
@@ -7,23 +7,38 @@
 // Write a function that takes zero or more arguments
 // and returns the minimum from that set
 // assume that all the argument are numbers
-const min = function min () {
+const min = function min (a) {
+  let result = aguments[0]
 
+  for(let i = 0; i <aruments.length; i++){
+    if result < arguments[i]) {
+      result =arguments[i];
+    }
+  }
+  return result;
 }
 
 // Question 2
 // Write a function that takes zero or more arguments
 // and returns the sum of that set
 // assume that all the argument are numbers
-const sum = function sum () {
-
+const sum = function sum (num1, num2) {
+  return  num1 + num2
```

Comment:
Yes, but the function should be able to accept an unlimited number of arguments. Similar to the function above ^^